docs/narr/MyProject/development.ini

```diff
@@ -1,6 +1,6 @@
 ###
 # app configuration
-# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
```

Comment:
Are we planning to keep these links up to date with each new version of Pyramid? Is there a particular reason we don't want to link to latest?